fe/fe-core/src/main/java/com/starrocks/scheduler/Task.java

```diff
@@ -0,0 +1,93 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.scheduler;
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.common.io.Text;
+import com.starrocks.common.io.Writable;
+import com.starrocks.persist.gson.GsonUtils;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.Map;
+
+public class Task implements Writable {
+
+    @SerializedName("id")
+    private long id;
+
+    @SerializedName("name")
+    private String name;
+
+    @SerializedName("createTime")
+    private long createTime;
+
+    @SerializedName("dbName")
+    private String dbName;
```

Comment:
Is this field really necessary? I did not find its usage.

Reply:
This is actually required. Normal tasks are created with DB granularity. I will send you some product design documents.